src/app/workflow/launch-third-party/dialog/launch-to-codespace-dialog.component.html

```diff
@@ -0,0 +1,35 @@
+<h1 mat-dialog-title>Launch this {{ term }} to a GitHub Codespace?</h1>
```

Comment:
Codespaces, plural, here and everywhere? I wouldn't have noticed this if you hadn't updated your web service PR to use plural.

Reply:
The way I think of it, "GitHub Codespaces" is the name of the service, and a "GitHub Codespace" is an instance of the environment that the service spawns.  So, you can both "launch to GitHub Codespaces" and "launch to a GitHub Codespace''.  To me, the latter phrase feels more natural and specific here...